src/components/Todo/Todo.tsx

```diff
@@ -54,8 +70,17 @@ const Todo = ({
           onChange={handleChange}
         />
         <div className="content">
-          <span>Checkbox Label</span>
-
+          <input
+            className="content-input"
+            style={{
+              backgroundColor: getBackGroundColor(),
+            }}
+            type="text"
+            value={text?.toString()}
```

Comment:
```suggestion
value={text}
```
If the text type is string I don't think you need to parse the value toString.